## 72% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 12
Passed tests: 31
```
```
IDs of failed tests: 702809,702859,702732,703086,702894,702783,703503,703202,702786,702844,702731,702808 
```
<h3>Failed tests (12)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=17269&TR_CASE_IDS=702809,702859,702732,703086,702894,702783,703503,703202,702786,702844,702731,702808">Rerun failed tests</a></li><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233673">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233707">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `authors-for-reaction-5` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_group_chat.py:344: in test_group_chat_reactions
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Incorrect reactions count for member_1 after changing the reactions
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e844f8900c847f3b5c34b5b8beeb7c7?auth=c07adaa27b95877805f0a66e91fe4e8d#150">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ffdb512fe6f64eee9fc7c8ad3b410694?auth=f54d1c6a91771a1783c27265d0691503#99">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1ea7fa7fee794c4f9ac750b35f3cfebb?auth=36236b18bd0be219cfea6cc17665da33#79">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233682">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><p><blockquote><div>Device 3: Looking for a message by text: message from old member </div><div>Device 3: Looking for a message by text: message from new member </div></blockquote></p>

```
critical/chats/test_group_chat.py:442: in test_group_chat_offline_pn
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Messages PN was not fetched from offline
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e844f8900c847f3b5c34b5b8beeb7c7?auth=c07adaa27b95877805f0a66e91fe4e8d#309">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ffdb512fe6f64eee9fc7c8ad3b410694?auth=f54d1c6a91771a1783c27265d0691503#219">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1ea7fa7fee794c4f9ac750b35f3cfebb?auth=36236b18bd0be219cfea6cc17665da33#179">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233713">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] https://github.com/status-im/status-mobile/issues/17175]]</b><br/><br/></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233672">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233696">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'reply to image')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-1']/android.widget.TextView` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
critical/test_public_chat_browsing.py:759: in test_community_one_image_send_reply
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Reply message was not received by the sender
E    Image message reaction is not shown for the sender
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#220">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#230">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233704">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1028: in test_community_mark_all_messages_as_read
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 New messages counter is not shown in home > Community element
E    New messages counter is not shown in community channel element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#544">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#347">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233698">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><p><blockquote><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 1: Looking for a message by text: Hurray! unblocked </div></blockquote></p>

```
critical/test_public_chat_browsing.py:967: in test_community_contact_block_unblock_offline
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Hurray! unblocked was not received in public chat after user unblock! 
```

<b>[[Message can be missed after unblock: https://github.com/status-im/status-mobile/issues/16873]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#357">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#324">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233690">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div>Device 1: Getting preview message for link: https://github.com/status-im/status-mobile/pull/11707 </div></blockquote></p>

```
critical/test_public_chat_browsing.py:841: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    message.wait_for_element(60)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `1`: `PreviewMessage` by` xpath`: `//*[starts-with(@text,'https://github.com/status-im/status-mobile/pull/11707')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#296">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#301">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233683">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div># STEP: Checking that "bold text in asterics" is applied (**) in community channel</div><div>Device 2: Sending message '**bold text in asterics**' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1172: in test_community_markdown_support
    self.channel_2.send_message(message_to_send)
../views/chat_view.py:995: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b37b7a6b29fb4655a7f2d94726530405?auth=4f719740c8e5fb7565cba79c60049e1f#170">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3937b0b60bed499586bd05380d130fd4?auth=b2898daee389d385c3cdf9a765f61bde#102">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233680">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><p><blockquote><div># STEP: Invited member sends a message with a mention</div><div>Device 2: Sending message 'hi' </div></blockquote></p>

```
critical/test_public_chat_browsing.py:1081: in test_community_mentions_push_notification
    self.channel_2.send_message("hi")
../views/chat_view.py:995: in send_message
    self.chat_message_input.wait_for_element(wait_chat_input_sec)
../views/base_element.py:121: in wait_for_element
    raise TimeoutException(
 Device `2`: `ChatMessageInput` by` accessibility id`: `chat-message-input` is not found on the screen after wait_for_element 
```

<b>[[Issue with username in PN, issue #6 in 15500]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b37b7a6b29fb4655a7f2d94726530405?auth=4f719740c8e5fb7565cba79c60049e1f#143">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3937b0b60bed499586bd05380d130fd4?auth=b2898daee389d385c3cdf9a765f61bde#97">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233678">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//*[starts-with(@text,'test message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='message-status']/android.widget.TextView` </div><div>Device 2: `Text` is `Delivered` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1416: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message was not delivered after resending from offline
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c03ccd7a02fc4a76bab48cb327676aeb?auth=14ac002bba5473f356c832bc1edc1e8f#78">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c3aabfe326f945b18f9ad10d37e04cfb?auth=55cd5aa7c53680e5f81bf03f636bfc2b#78">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (31)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233711">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e844f8900c847f3b5c34b5b8beeb7c7?auth=c07adaa27b95877805f0a66e91fe4e8d#361">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ffdb512fe6f64eee9fc7c8ad3b410694?auth=f54d1c6a91771a1783c27265d0691503#244">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1ea7fa7fee794c4f9ac750b35f3cfebb?auth=36236b18bd0be219cfea6cc17665da33#219">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233708">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e844f8900c847f3b5c34b5b8beeb7c7?auth=c07adaa27b95877805f0a66e91fe4e8d#259">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ffdb512fe6f64eee9fc7c8ad3b410694?auth=f54d1c6a91771a1783c27265d0691503#177">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1ea7fa7fee794c4f9ac750b35f3cfebb?auth=36236b18bd0be219cfea6cc17665da33#132">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233681">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8e844f8900c847f3b5c34b5b8beeb7c7?auth=c07adaa27b95877805f0a66e91fe4e8d#127">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ffdb512fe6f64eee9fc7c8ad3b410694?auth=f54d1c6a91771a1783c27265d0691503#72">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1ea7fa7fee794c4f9ac750b35f3cfebb?auth=36236b18bd0be219cfea6cc17665da33#72">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233705">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9f2da9c7f404e818ae745f793bf4da8?auth=25839d58e66d4d0db9f0c4e740548c6f#254">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233675">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9f2da9c7f404e818ae745f793bf4da8?auth=25839d58e66d4d0db9f0c4e740548c6f#75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233697">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9f2da9c7f404e818ae745f793bf4da8?auth=25839d58e66d4d0db9f0c4e740548c6f#104">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233692">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9f2da9c7f404e818ae745f793bf4da8?auth=25839d58e66d4d0db9f0c4e740548c6f#54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233709">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e9f2da9c7f404e818ae745f793bf4da8?auth=25839d58e66d4d0db9f0c4e740548c6f#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233706">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#195">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#161">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233687">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#273">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#286">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233686">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#188">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233685">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#146">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233689">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233688">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ce5b60a1a8e6422689d7d3cd02c7b04a?auth=6601e849a8138397e4d91d82a8e14629#312">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f361e145830b4aa9807ac4dbd5ca766c?auth=7a01631c1d91f8690483740c4c7747b1#314">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233702">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/50cd9df16f41412fb316cd5afee16639?auth=e034282d1fb3005b68fcf77b4aaa542e#117">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3e7479b5ee9943508152e662343cde68?auth=14040baa5dac97befadcdc45faa61ab3#96">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233703">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/50cd9df16f41412fb316cd5afee16639?auth=e034282d1fb3005b68fcf77b4aaa542e#172">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3e7479b5ee9943508152e662343cde68?auth=14040baa5dac97befadcdc45faa61ab3#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233701">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b37b7a6b29fb4655a7f2d94726530405?auth=4f719740c8e5fb7565cba79c60049e1f#199">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3937b0b60bed499586bd05380d130fd4?auth=b2898daee389d385c3cdf9a765f61bde#191">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233691">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b37b7a6b29fb4655a7f2d94726530405?auth=4f719740c8e5fb7565cba79c60049e1f#196">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3937b0b60bed499586bd05380d130fd4?auth=b2898daee389d385c3cdf9a765f61bde#146">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233699">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c07be4eaa6f4bf784b5975f61a15138?auth=53445119e8db8c931c85afbdc9a69efd#141">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/634c24ff581d462d879ae56cff00bc87?auth=dbf1446c378270ddb6be852cb94843ce#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233700">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c07be4eaa6f4bf784b5975f61a15138?auth=53445119e8db8c931c85afbdc9a69efd#166">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/634c24ff581d462d879ae56cff00bc87?auth=dbf1446c378270ddb6be852cb94843ce#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233677">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#130">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233674">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#413">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#347">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233684">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#200">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233676">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#192">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#150">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233695">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#326">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#238">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233710">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#353">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#280">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233671">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/eeead38dcba84f0cb5c1136da5683917?auth=2df5cee6f7dc6e60a5be78a625522118#80">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f76c4f81f427463eb115985a4d0562ce?auth=62dfa3c964e61c7bd8a00dd7de1d5151#78">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233694">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c166a123b4b4e4c9c2b685ca4382418?auth=4abcb09fdde74a5061595c77c166ea57#91">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96537f9a645d4534be24866d0bc83d09?auth=8cf22cf5b756adffd9d10612bf4ee0a7#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233693">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c166a123b4b4e4c9c2b685ca4382418?auth=4abcb09fdde74a5061595c77c166ea57#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/96537f9a645d4534be24866d0bc83d09?auth=8cf22cf5b756adffd9d10612bf4ee0a7#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233679">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c03ccd7a02fc4a76bab48cb327676aeb?auth=14ac002bba5473f356c832bc1edc1e8f#205">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c3aabfe326f945b18f9ad10d37e04cfb?auth=55cd5aa7c53680e5f81bf03f636bfc2b#211">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3233712">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c03ccd7a02fc4a76bab48cb327676aeb?auth=14ac002bba5473f356c832bc1edc1e8f#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c3aabfe326f945b18f9ad10d37e04cfb?auth=55cd5aa7c53680e5f81bf03f636bfc2b#198">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im2/contexts/onboarding/common/background/view.cljs
+++ b/src/status_im2/contexts/onboarding/common/background/view.cljs
@@ -26,9 +26,10 @@
 (defn background-image
   [content-width]
   [rn/image
-   {:style  {:resize-mode :stretch
-             :margin-top  32
-             :width       content-width}
+   {:style  {:resize-mode   :stretch
+             :resize-method :scale
+             :margin-top    32
+             :width         content-width}
     :source (resources/get-image :onboarding-illustration)}])
 
 (defonce progress (atom nil))

--- a/src/status_im2/contexts/onboarding/common/carousel/style.cljs
+++ b/src/status_im2/contexts/onboarding/common/carousel/style.cljs
@@ -64,7 +64,6 @@
 (defn carousel-container
   [left animate?]
   (cond->> {:position       :absolute
-            :right          0
             :top            0
             :flex-direction :row
             :left           left}

